frontend/package.json

```diff
@@ -39,6 +39,7 @@
         "array-move": "^4",
         "framer-motion": "^7",
         "js-base64": "^3.7.5",
+        "lodash-es": "^4.17.21",
```

Comment:
Not worth adding a dependency for just isEqual.

There are 3 options that are better:

You can either iterate over `Object.entries` and compare property by property, or use mobx `comparer.structural` (which essentially does the same thing, but also nesting), or simply write a custom isEqual function,

Reply:
@rikimaru0345 done